### ISSUE TITLE
"Supplemental guidance" or another term?

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-wcag-supplemental#39]
Context: We are close to publishing the COGA objectives and design patterns on the WAI website along with redesigned Understanding WCAG, Techniques, and (ACT) Test Rules. We hope soon to also publish some "supplemental guidance" on low vision accessibility, and maybe other areas, too, e.g., mobile.

See **[in-progress prototype](https://wai-wcag-supplemental.netlify.app/wcag-supplemental/all-supplemental-guidance/)**.

"supplemental guidance" was the term used in WCAG 2.1:
> We encourage authors to refer to our **supplemental guidance** on [improving inclusion for people with disabilities, including learning and cognitive disabilities, people with low-vision, and more](http://www.w3.org/WAI/standards-guidelines/wcag/#supplement).

However, "supplemental" is not a simple word. And is "guidance" too close to "guidelines"? If we are going to call this something different, now is the time to do it.

(We can't change 2.1. And we will leave that target link. We could change that sentence in WCAG 2.2.)

Thoughts on alternatives to "supplemental guidance"?

**Comment by bakkenb** [comment]
I don't have an overwhelming issue with using the term "supplemental guidance." I don't see the use of the word guidance being misconstrued with "guidelines."

If not too much of an understanding barrier with non-native English speakers, the terms "Ancillary Guidance" or "Ancillary Support" would be an acceptable alternative. 

The word ancillary "picks up on the notion of providing aid or support in a way that supplements something else. In particular, the word often describes something that is in a position of secondary importance." -Webster. I just do not know if the word ancillary would be problematic for others.

**Comment by carlosapaduarte** [comment]
Non-native speaker here.

I agree with Brent regarding "guidance". I don't consider that to be an issue. 

While I'm comfortable with "supplemental" (I'm used to it), and I don't have an issue with "ancillary" (although it's more uncommon than the alternatives), I do believe "additional" is the simplest of the 3 options proposed so far. It was the alternative my mind wondered to before reaching Shawn's initial proposal. 

If we consider the [search trends for the 3 options](https://trends.google.com/trends/explore?q=additional,supplemental,ancillary), it's clear that "additional" is more commonly used in searches than the other two.

Considering familiarity with the content, I would keep "supplemental", which would be better for existing "users" of W3C resources. Considering the simplicity of the content, my vote goes for "additional", which is probably better for people new to these resources.

**Comment by SteveALee** [comment]
-1 to ancillary
+1 to additional
+1 to guidance  (though I note guidelines is a bit weak given the status of WCAG)

**Comment by sharronrush** [comment]
-1 to ancillary for the reasons stated
Not sure about guidance, I can see how there may be confusion about guidelines/guidance and which are required vs recommended. Easier for us to know that difference, being steeped in it. 

What about another approach? If we emphasize that the content of the "additional" information is to meet the specific needs of certain users, might that not be clearer about the intent and status of the advice?

To me those would be clearer if the title became something like "Meeting User Needs" for brevity -  or "Additional Techniques to Meet User Needs," if required to be more definitive.

**Comment by shawna-slh** [comment]
Note that right now there are sections for COGA and Low Vision. In the future there may be sections for mobile or other specific situations. That is, the sections could be something other than user groups.

I think we want to avoid words that have a specific meaning in WCAG and W3C, e.g., "techniques" and "recommendations"

Also, btw, there are reasons not not use to "best practice". (though I cannot articulate them well now)

**Comment by shawna-slh** [comment]
A couple points.

First, please see the context in which this is used, e.g., header at https://wai-wcag-supplemental.netlify.app/wcag-supplemental/all-supplemental-guidance/
> Supplemental Guidance to WCAG 2

Second, an important point is communicating that this is not required to meet WCAG. We have this wording a couple of places:
* Supplemental Guidance is informative, not required to meet WCAG.
* Supplemental guidance provides advice… on improving the accessibility of content and applications beyond what is required by Web Content Accessibility Guidelines (WCAG).

It would be good if it is also communicated in the header text.

“Supplemental Guidance to WCAG 2” is shorter version of “guidance that is supplemental to WCAG 2”. However, “Additional Guidance to WCAG 2” doesn’t seem as smooth.
Perhaps:
“**Additional Guidance Beyond WCAG 2**”
?

**Comment by AndrewArch** [comment]
I do agree that 'additional' seems simpler than 'supplemental' [supplemental = provided in addition to what is already present or available to complete or enhance it] and better than 'ancillary'.

What about 'complementary guidance' as it complements and adds to WCAG?

**Comment by SteveALee** [comment]
One disadvantage to additional (and possibly complementary) is that while it accurately describes the relationship to WCAG, especially for legal reasons, it undervalues the importance for an inclusive user experience!

**Comment by lauracarlson** [comment]
As WCAG is a baseline, +1 to something such as "Best Practices: Beyond WCAG".

**Comment by mraccess77** [comment]
Supplemental guidance is good with me.

**Comment by bruce-usab** [comment]
Of the suggestions in this thread, <q>Additional Guidance Beyond WCAG 2</q> resonates for me.

> Also, btw, there are reasons not to use "best practice". (though I cannot articulate them well now)

I would like to know more about these reasons!  On concern I have heard (in a different context) is that this sort of guidance might more accurately be characterized as _better_ practices &mdash; but probably not the _best_ practices.  Also, since 2.0 Level AA is the baseline (which is an objectively a good thing), it is also objectively (for many settings) the absolute bare minimum.  Characterizing something that does a little bit more than the bare minimum as a <q>best practice</q> seems hyperbolic.

**Comment by lauracarlson** [comment]
+1

**Comment by SteveALee** [comment]
I'd like to think the guidance would eventually move into to WCAG so the status should not implied to be permanent :)

**Comment by jade-mc** [comment]
I'd vote for additional resources/guidance. 
Any form of supplement is really high level in terms of their language (using the [EVP corpus](url) as reference) it's coming out as a C2 level which indicates low frequency, native speaker level, whereas additional is a B2 (the scale goes from A0-C2 in terms of difficulty.) 
Additional resources/guidance is all B2 and much more accessible to a global audience.

**Comment by Myndex** [comment]
### Designing for Accessibility:
**_Reaching Beyond Minimum Guidelines      
for an Improved User Experience._**

**Comment by sharronrush** [comment]
Ummm, no one seemed to prefer supplemental although several said they "had no problem with it." Not sure why this is closed?

**Comment by shawna-slh** [comment]
COGA Facilitators, LVTF Facilitator, AG WG Chairs, and EOWG Chairs were asked to "share this GitHub issue with your groups as appropriate" on 13 January.

Here are [EOWG minutes from 21 January 2022.](https://www.w3.org/2022/01/21-eo-minutes#item02)

**Comment by shawna-slh** [comment]
After consideration, there was not a replacement that didn't have issues.

Therefore, we plan to leave the term as was agreed upon with COGA folks and WCAG folks a few years ago, and not spend more time debating it and re-negotiating consensus.
